src/main/java/catan/domain/transfer/output/game/GameUserDetails.java

```diff
@@ -10,9 +11,12 @@
     private boolean ready;
     private int moveOrder;
     private AchievementsDetails achievements;
+    private AllAvailableActionsDetails availableActions;
     private ResourcesDetails resources;
     private DevelopmentCardsDetails developmentCards;
 
+    private static final Gson GSON = new Gson();
```

Comment:
nor very good idea to have such util constants placed in DTOs, maybe call it from some service or util class?